end2end/tests/adminPanelWorkflowEditor-eventsEditor.spec.ts

```diff
@@ -436,124 +448,69 @@ test.describe('Email events can be used to send or CC based on orgchart_group re
     await expect(editorPage.getByRole('textbox', { name: 'Email To:' })).toBeVisible();
-    await editorPage.getByRole('textbox', { name: 'Email To:' }).fill(emailToCC);
+    await editorPage.getByRole('textbox', { name: 'Email To:' }).fill(emailTo);
+    await editorPage.getByRole('textbox', { name: 'Email CC:' }).fill(emailCC);
 
     let subjectArea = editorPage
       .locator('#divSubject')
-      .getByLabel('Template Editor coding area.')
-
+      .getByLabel('Template Editor coding area.');
     await subjectArea.press('ControlOrMeta+A');
     await subjectArea.press('Backspace');
     await subjectArea.fill(subjectText);
 
+    await editorPage.getByRole('button', { name: 'Use Code Editor' }).click();
+    
+    let bodyArea = editorPage
+      .locator('#code_mirror_template_editor');
+    await bodyArea.press('ControlOrMeta+A');
+    await bodyArea.press('Backspace');
+    await bodyArea.fill(bodyContent); 
+       
     await editorPage.getByRole('button', { name: 'Save Changes' }).click();
-  });
 
-  test('Trigger event with email recipient based on request orgchart_group data', async({page}) => {
+    //Trigger event with Return to Requestor workflow action
     await page.goto(requestURL);
     await page.waitForLoadState('load');
-
-    await page.locator("#PHindicator_9_1 button").click();
-    const dialog = page.getByRole('dialog', { name: 'Editing #' });
-    await expect(dialog.locator('div[id$="_loadIndicator"]')).toBeHidden();
-
-    await dialog.getByRole('searchbox', { name: 'Search for user to add as' }).fill(groupId);
-    await dialog.getByRole('cell', { name: groupDisplayName }).click();
-
-    await expect(page.getByRole('searchbox', { name: 'Search for user to add as' })).toHaveValue(groupId);
-
-    await awaitPromise(page, 'getprintindicator', async (d:Locator) => {
-      await d.getByRole('button', { name: 'Save Change' }).click();
-    });
-
-    //Trigger event with Return to Requestor workflow action
     await expect(page.getByRole('button', { name: 'Return to Requestor' })).toBeVisible();
     await page.getByRole('button', { name: 'Return to Requestor' }).click();
   });
 
-
-  test('Verify Email Sent (To)', async({page}) => {
+  test('Verify email sent, email recipients (To/Cc/notify), and email field content)', async({page}) => {
     await page.goto('http://host.docker.internal:5080/');
     await page.waitForLoadState('load');
-    await confirmEmailRecipients(page, subjectText, expectedFieldEmailRecipients);
+    await confirmEmailRecipients(page, subjectText, expectedToFieldEmailRecipients);
+    await confirmEmailRecipients(page, subjectText, expectedCcFieldEmailRecipients);
     await confirmEmailRecipients(page, subjectText, expectedEventEmailRecipients);
 
-    //Cleanup the inbox
     await page.getByText(subjectText).first().click();
+       
+    const gridLocator = page.locator('iframe').contentFrame();
+    await expect(
+       gridLocator.getByRole('table'),
+    'The Grid is present'

```

Comment:
Can you get this to work with the '#format_grid' id set in the content div?  That way we're more certain that it's the table for the grid that was found and not some other table in the iframe.